pkg/virt-handler/devices/mandatory.go

```diff
@@ -0,0 +1,103 @@
+package devices
```

Comment:
This file contains now the setup for the two mandatory plugins `/dev/net/tun` and `/dev/kvm`.